packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -403,8 +404,19 @@ export class FlexFieldSchema<
 				monomorphicChildType: schema !== Any ? oneFromSet(schema) : undefined,
 			};
 		});
+
+		this.lazyTypes = lazy;
+
+		this.stored = {
+			kind: this.kind.identifier,
+			get types() {
```

Comment:
nit: if you make this a fat-arrow then you can just use `this.lazyTypes` and don't have to factor out `lazy` into a local above.

Reply:
Thats not permitted in the JS grammar: https://stackoverflow.com/questions/33827519/es6-getter-setter-with-arrow-function

Reply:
Huh, I guess not. I thought it was. Interesting.